src/core/sys/posix/sys/types.d

```diff
@@ -418,7 +423,7 @@ else version( CRuntime_Bionic )
     alias uint     id_t;
     alias int      key_t;
     alias c_long   suseconds_t;
-    alias c_long   useconds_t;
+    alias uint     useconds_t; // Updated in Lollipop
```

Comment:
So the bindings will only ever support one version of Android?

Reply:
No, I believe every version since uses this now, just noting the version where the switch was made.